src/textfield.vue

```diff
@@ -92,6 +94,16 @@ export default {
   methods: {
     fireInputEvent: function (event) {
       this.$emit('input', event.target.value)
+    },
+    focus () {
+      this.$refs.focusTarget.focus()
+    },
+    focusAndSelect () {
```

Comment:
I started with just the `focus()` call, but then realised in practice you would want to both focus _and_ select any existing text. I didn't actually check if calling the underlying `select()` method would also focus the input element, and I guess that _could_ be different in different browsers? In any case, it seemed you might want either 'focus but no select' or 'focus and select'. The primary reason for these is to move the input focus, which is why I wanted to have 'focus' in both method names. Just calling it `select()` doesn't suggest it would also move the input focus, which might be confusing.